pkg/agent/manager/manager.go

```diff
@@ -39,7 +39,7 @@ type Manager interface {
 
 	// SubscribeToBundleChanges returns a new observer.Stream on which []*x509.Certificate instances are
```

Comment:
it is no longer returning a observer.Stream

Reply:
👍